modules/nw-configuring-sriov-in-app-namespace.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-configuring-sriov-in-app-namespace_{context}"]
+= Configuring SriovNetwork in application namespaces
+
+When an SriovNetwork custom resource (CR) is deployed in an application namespace, do not define or populate the `spec.networkNamespace` field. In this scenario, the NetworkAttachmentDefinition will be created in the same namespace as the SriovNetwork CR.
+
+The sriov-operator webhook will reject the creation of an `SriovNetwork` resource in an application namespace if the spec.networkNamespace field is defined.
+
+Follow this procedure to create an SriovNetwork resource in an application namespace and attach a pod to the additional network.
+
+.Prerequisites
+
+The following steps must be completed by a cluster administrator before an application owner can configure a namespaced SriovNetwork resource: 
+
+* The SR-IOV Network Operator is installed in the  openshift-sriov-network-operator namespace.
+* Nodes with SR-IOV hardware are labeled for the operator to identify the nodes.
+*
+
+As an application owner you need to have administrator privileges on the application namespace.
+
+.Procedure
+
+. Specify the SR-IOV network device configuration for a node by creating an SR-IOV network node policy. The `SriovNetworkNodePolicy` object is created in the `openshift-sriov-network-operator` namespace to define the SR-IOV network device configuration for nodes. Example configuration for Intel DPK is as follows: 
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: intel-dpdk-node-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: intelnics
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  priority: 10
+  numVfs: 4
+  nicSelector:
+    vendor: "8086"
+    deviceID: "158b"
+    pfNames: [""]
+  deviceType: vfio-pci
```

Comment:
`vfio-pci` value is good, but if the user wants to see something in the verification step `oc exec -it test-pod -n sriov-app -- ip a` , then we need `netdevice`